index.php

```diff
@@ -1,9 +1,13 @@
 <?php
 $links = array(
+    'blog' => 'https://tcmaker.org/news',
```

Comment:
@automatingerror You can comment on specific lines of code.

Reply:
Is the Facebook Group the only Facebook resource we will be linking with the shortener? If it isn't we may want to make the link more descriptive.

Reply:
For now, it is. I think `fb` should point to our main Facebook presence, whether that's the group or the page. I don't care which one it is, so long as we pick one and stick with it.